main.go

```diff
@@ -46,19 +55,26 @@ func main() {
 		}
 		os.Exit(0)
 	case "dispense":
-		expt = dispense.Parse(os.Args[2:])
+		for i, arg := range os.Args {
+			if arg == "dispense" && i+1 < len(os.Args) {
+				dspsArgs = os.Args[i+1:]
```

Comment:
`dspsArgs` and `dtbsArgs` looks so similar that it's hard to see the difference at first glance. It is better to not use acronyms like these for code readability.